src/test/resources/configuration-as-code.yml

```diff
@@ -0,0 +1,39 @@
+unclassified:
+  email-ext:
+    defaultSuffix: "@domain.extension"
+    addAccounts:
+      - address: "foo.bar"
+        smtpHost: "smtp-host"
+        smtpPort: "1234"
+        smtpUsername: "smtp-username"
+        smtpPassword: "smtp-password"
+        useSsl: true
+        advProperties: "avd-properties"
+    charset: "UTF-8"
+    defaultContentType: "text/html"
+    defaultSubject: "DEFAULT EMAIL SUBJECT"
+    defaultBody: "DEFAULT BODY"
+    emergencyReroute: "emergency-reroute"
+    maxAttachmentSize: 42
+    mailAccount:
+      address: "foo.bar.xyz"
```

Comment:
The address field should not be set for the `mailAccount` member.

Reply:
noted. I'll remove it. Even if it can be configured, not sure of the side effects if it is..

Reply:
There shouldn't be any side effects that I can think of, just don't want people to look at it as an example.

Reply:
> just don't want people to look at it as an example.

I totally agree.
I just put it there, looking at the classes and what was possible to configure there each time.